wiki/Getting-Started-UIKit.md

```diff
@@ -0,0 +1,229 @@
+
+```swift
+import UIKit
+import Workflow
+import WorkflowUIKit
+
+class ViewController: UIViewController {
+    private let launchButton = UIButton()
+
+    override func viewDidLoad() {
+        launchButton.setTitle("Launch Workflow", for: .normal)
+        launchButton.setTitleColor(.systemBlue, for: .normal)
+        launchButton.addTarget(self, action: #selector(didTapLaunchWorkflow), for: .touchUpInside)
+
+        view.addSubview(launchButton)
+
+        launchButton.translatesAutoresizingMaskIntoConstraints = false
+        launchButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        launchButton.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+    }
+
+    @objc private func didTapLaunchWorkflow() {
+        let workflow = Workflow(FirstViewController.self)
+            .thenPresent(SecondViewController.self)
+
+        launchInto(workflow, args: "Noble Six") { passedArgs in
+            workflow.abandon()
+
+            guard case .args(let emailAddress as String) = passedArgs else {
+                print("No email address supplied")
+                return
+            }
+            print(emailAddress)
+        }
+    }
+}
+```
+### Let's discuss what's going on here.
+#### **Where is the type safety, I heard about?**
+<details>
+
+The [Workflow](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/Workflow.html) has compile-time type safety on the Input/Output types of the supplied [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html)s. This means that you will get a build error if the output of `FirstViewController` does not match the input type of `SecondViewController`.
+</details>
+
+#### **What's going on with this `passedArgs`?**
+<details>
+
+The `onFinish` closure for `launchInto(_:args:onFinish:)` provides the last passed [AnyWorkflow.PassedArgs](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/AnyWorkflow/PassedArgs.html) in the work flow. For this Workflow, that could be the output of `FirstViewController` or `SecondViewController` depending on the email signature typed in `FirstViewController`. To extract the value, we unwrap the variable within the case of `.args()` as we expect this workflow to return some argument.
+</details>
+
+#### **Why call `abandon()`?**
+<details>
+
+Calling `abandon()` closes all the views launched as part of the workflow, leaving you back on `ViewController`.
+</details>
+
+## Testing
+
+### Installing test dependencies
+
+For our test example, we will be using a library called [UIUTest](https://github.com/nallick/UIUTest). It is optional for testing Workflow, but in order for the example to be copyable, you will need to add the UIUTest Swift Package
+to your test target.
```

Comment:
Wondering about calling out CocoaPods instructions for UIUTest, but I feel that with the link to the repo, a CocoaPods user will be able to find out how to include it.

Reply:
I agree, I think we can leave this as is.